conf/nginx.conf.tpl

```diff
@@ -26,6 +26,11 @@ http {
   upstream web { server unix:/tmp/gunicorn-web.sock fail_timeout=0; }
   upstream websocket { server unix:/tmp/gunicorn-websocket.sock fail_timeout=0; }
 
+  map $http_hypothesis_test_migration $root_url {
```

Comment:
`$root_url` seems like a bit of an odd name for this. Maybe `$default_upstream` would make more sense?